### PR TITLE
Use silent mapping

### PR DESCRIPTION
**Pull request by wsdjeg**
Fix https://github.com/Shougo/dein.vim/issues/98

### CODE REVIEW

File: plugin/ctrlsf.vim [hunk 1]
```diff
@@ -219,21 +219,21 @@ com! -n=0                                         CtrlSFToggle  call ctrlsf#Togg
...
-nnoremap <expr> <Plug>CtrlSFPwordExec <SID>SearchPwordCmd('CtrlSF', 1)
-
-nnoremap        <Plug>CtrlSFQuickfixPrompt :CtrlSFQuickfix<Space>
-nnoremap <expr> <Plug>CtrlSFQuickfixCwordPath <SID>SearchCwordCmd('CtrlSFQuickfix', 0)
-nnoremap <expr> <Plug>CtrlSFQuickfixCwordExec <SID>SearchCwordCmd('CtrlSFQuickfix', 1)
-vnoremap <expr> <Plug>CtrlSFQuickfixVwordPath <SID>SearchVwordCmd('CtrlSFQuickfix', 0)
-vnoremap <expr> <Plug>CtrlSFQuickfixVwordExec <SID>SearchVwordCmd('CtrlSFQuickfix', 1)
-nnoremap <expr> <Plug>CtrlSFQuickfixPwordPath <SID>SearchPwordCmd('CtrlSFQuickfix', 0)
-nnoremap <expr> <Plug>CtrlSFQuickfixPwordExec <SID>SearchPwordCmd('CtrlSFQuickfix', 1)
+nnoremap <silent> <Plug>CtrlSFPrompt    :CtrlSF<Space>
```
Comment (dyng) [inline plugin/ctrlsf.vim]: Appending `<silent>` does not make sense for `<Plug>CtrlSFPrompt`, this map is used for waiting your further input, `<silent>` makes nothing visible for user and can be confusing.

**Comment by dyng** [comment]
Thanks for your pull request, `<silent>` is a meanful change but not suitable for all mappings, it makes sense for only **execution** mapping, please fix that. Thanks.

- CtrlSFCwordExec
- CtrlSFVwordExec
- CtrlSFPwordExec
- CtrlSFQuickfixCwordExec
- CtrlSFQuickfixVwordExec
- CtrlSFQuickfixPwordExec

File: plugin/ctrlsf.vim (hunk 1 above)
Comment (wsdjeg) [inline plugin/ctrlsf.vim]: yeah,thanks,I will fix it tonight.

File: plugin/ctrlsf.vim [hunk 2]
```diff
@@ -48,7 +48,7 @@ endf
 " s:SearchCwordCmd() {{{2
 func! s:SearchCwordCmd(type, to_exec)
     let cmd = ":\<C-U>" . a:type . " " . expand('<cword>')
-    let cmd .= a:to_exec ? "<CR>" : " "
```
Comment (wsdjeg) [inline plugin/ctrlsf.vim]: http://blog.csdn.net/boyinnju/article/details/6877087
I am not sure,if we can use `<CR>` instead.

File: plugin/ctrlsf.vim [hunk 3]
```diff
@@ -58,16 +58,16 @@ endf
...
+    let cmd = ":\<C-U>call feedkeys(" . keys . ", 'n')\<CR>"
     return cmd
 endf
 " }}}
 
 " s:SearchPwordCmd() {{{2
 func! s:SearchPwordCmd(type, to_exec)
     let cmd = ":\<C-U>" . a:type . " " . @/
-    let cmd .= a:to_exec ? "\r" : " "
+    let cmd .= a:to_exec ? "\<CR>" : " "
```
Comment (dyng) [inline plugin/ctrlsf.vim]: "\r" and "\<CR>" is **identical** (you can verify it by running `:echo "\r" ==# "\<CR>"`, i'm not sure what is this change for?

File: plugin/ctrlsf.vim [hunk 4]
```diff
@@ -219,21 +219,21 @@ com! -n=0                                         CtrlSFToggle  call ctrlsf#Togg
...
+nnoremap <Plug>CtrlSFQuickfixPrompt :CtrlSFQuickfix<Space>
 nnoremap <expr> <Plug>CtrlSFQuickfixCwordPath <SID>SearchCwordCmd('CtrlSFQuickfix', 0)
-nnoremap <expr> <Plug>CtrlSFQuickfixCwordExec <SID>SearchCwordCmd('CtrlSFQuickfix', 1)
+nnoremap <silent> <expr> <Plug>CtrlSFQuickfixCwordExec <SID>SearchCwordCmd('CtrlSFQuickfix', 1)
 vnoremap <expr> <Plug>CtrlSFQuickfixVwordPath <SID>SearchVwordCmd('CtrlSFQuickfix', 0)
-vnoremap <expr> <Plug>CtrlSFQuickfixVwordExec <SID>SearchVwordCmd('CtrlSFQuickfix', 1)
+vnoremap <silent> <expr> <Plug>CtrlSFQuickfixVwordExec <SID>SearchVwordCmd('CtrlSFQuickfix', 1)
 nnoremap <expr> <Plug>CtrlSFQuickfixPwordPath <SID>SearchPwordCmd('CtrlSFQuickfix', 0)
-nnoremap <expr> <Plug>CtrlSFQuickfixPwordExec <SID>SearchPwordCmd('CtrlSFQuickfix', 1)
+nnoremap <silent> <expr> <Plug>CtrlSFQuickfixPwordExec <SID>SearchPwordCmd('CtrlSFQuickfix', 1)
```
Comment (dyng) [inline plugin/ctrlsf.vim]: hm... not a big problem but can you do me a favor to indent these lines? thanks.

File: plugin/ctrlsf.vim (hunk 3 above)
Comment (wsdjeg) [inline plugin/ctrlsf.vim]: I know ,just make the code more readable ,as we know ,most people use `<CR>` for `Enter` command.maybe it is the first time I see someone use `\r` for `Enter`.

File: plugin/ctrlsf.vim (hunk 4 above)
Comment (wsdjeg) [inline plugin/ctrlsf.vim]: no problem

File: plugin/ctrlsf.vim (hunk 3 above)
Comment (dyng) [inline plugin/ctrlsf.vim]: ok, `\<CR>' should be also fine.

**Comment by wsdjeg** [comment]
Done

**Comment by dyng** [comment]
Looks good to me. Thanks for your contribution!

**Pull request merged**